internal/check-imports/src/references.ts

```diff
@@ -63,7 +63,7 @@ log(runCliAction);
  * address
  */
 log(Address);
-log(BaseAssetId);
+log(ZeroBytes32);
```

Comment:
We should remove this log as it does not have a purpose anymore

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1976/commits/cef28a387a71fc2615fb6112bb6466be98e81acd